forc/src/ops/forc_abi_json.rs

```diff
@@ -72,13 +73,13 @@ pub fn build(command: JsonAbiCommand) -> Result<Value, String> {
             // Download a non-local dependency if the `git` property is set in this dependency.
             if let Some(git) = &dep.git {
                 let fully_qualified_dep_name = format!("{}-{}", dependency_name, git);
-                let downloaded_dep_path = match dependency::download_github_dep(
+                let downloaded_dep_path = match block_on(dependency::download_github_dep(
```

Comment:
you can make these functions `async` as well - just be careful to correctly handle the recursion since some of them are recursive,

if possible avoid the recursion
if not possible there are ways to get around it